pipelines/managed/push-disk-images-to-marketplaces/push-disk-images-to-marketplaces.yaml

```diff
@@ -35,10 +35,6 @@ spec:
         Extra rule data to be merged into the policy specified in params.enterpriseContractPolicy. Use syntax
         "key1=value1,key2=value2..."
       default: "pipeline_intention=release"
-    - name: enterpriseContractTimeout
```

Comment:
I see this one didn't have the worker count param. I wonder if it would make sense to just add it to the conforma task call to make it consistent with the others? Because these data fields will get documented and then users will expect them to always work - unless we also document that it only works for some pipelines.

Reply:
Actually this one and release-to-mrrc

Reply:
OK